### DISCUSSION TITLE
add SQLite3MultipleCiphers lib support

### DISCUSSION THREAD

**Discussion by shamork** [ericsink/SQLitePCL.raw#513] [Ideas]
SQLite3MultipleCiphers
[https://github.com/utelle/SQLite3MultipleCiphers/](https://github.com/utelle/SQLite3MultipleCiphers)

I've list steps to to use SQLite3MultipleCiphers with SQLitePCL.raw in [https://github.com/shamork/SQLitePCLRaw.multipleciphers](https://github.com/shamork/SQLitePCLRaw.multipleciphers).

But I think I do not understand the this project structure enough to make a PR.

**Comment by ericsink** [comment]
Yeah, adding providers and a bundle is not trivial.

One question I have is whether I would even want to accept responsibility for maintaining support for SQLite3MultipleCiphers.  I already maintain `e_sqlcipher`, the hassle is high enough that very often I wish I did not.  :-)
